Examples/ObjectiveC/DDSCircleLayerExample.m

```diff
@@ -0,0 +1,62 @@
+#import "DDSCircleLayerExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleDDSCircleLayer = @"DDSCircleLayerExample";
+
+@interface DDSCircleLayerExample () <MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation DDSCircleLayerExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    // Create a new map view using the Mapbox Light style.
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds
+        styleURL:[MGLStyle lightStyleURLWithVersion:9]];
+    
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    
+    // Set the map’s center coordinate and zoom level.
+    self.mapView.centerCoordinate = CLLocationCoordinate2DMake(38.897,-77.039);
+    self.mapView.zoomLevel = 10.5;
+    
+    self.mapView.delegate = self;
+    [self.view addSubview: self.mapView];
+}
+
+// Wait until the style is loaded before modifying the map style.
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    MGLSource *source = [[MGLVectorSource alloc] initWithIdentifier:@"trees" configurationURL:[NSURL URLWithString:@"mapbox://examples.2uf7qges"]];
+    
+    [self.mapView.style addSource:source];
+    
+    MGLCircleStyleLayer *layer = [[MGLCircleStyleLayer alloc] initWithIdentifier:[NSString stringWithFormat:@"tree-style"] source:source];
+    
+    // The source name from the source's TileJSON metadata: mapbox.com/api-documentation/#retrieve-tilejson-metadata
+    layer.sourceLayerIdentifier = @"yoshino-trees-a0puw5";
+    
+    // Categorical stops based on age of tree in years.
```

Comment:
I don't think that this would be a categorical stop, since you're using `MGLInterpolationModeInterval`. Maybe you could say "stops" or "stops dictionary?"